control/responsible_role.go

```diff
@@ -0,0 +1,84 @@
+package control
+
+import (
+	"github.com/jbowtie/gokogiri/xml"
+	"errors"
+	"regexp"
+	"strings"
+	"fmt"
+)
+
+// FindResponsibleRole looks for the Responsible Role cell in the control table.
+func FindResponsibleRole(ct *Table) (*ResponsibleRole, error) {
+	nodes, err := ct.searchSubtree(".//w:tc//w:t[contains(., 'Responsible Role')]")
+	if err != nil {
+		return nil, err
```

Comment:
I do. 
The reason I do is because of shadowing variables (which actually doesn't apply to return named values, but it does for other cases)

example of those other cases https://play.golang.org/p/fQSP1WxjaT

Go is smart enough to not shadow a return value. But those other cases have just influenced how I code. Error fast and return quickly explicitly.